src/components/BookList.js

```diff
@@ -1,21 +1,99 @@
 import Book from './Book'
+import styled from 'styled-components/macro'
+import { useState } from 'react'
 
 function BookList({ books }) {
+  const [filteredBooks, setFilteredBooks] = useState(books)
+  const [readingStatus, setReadingStatus] = useState('')
+
+  function handleActiveReadingStatus(status, books) {
+    setReadingStatus(status)
+    if (status === 'true') {
+      const readBooks = books.filter(book => book.finished === true)
+      setFilteredBooks(readBooks)
+    } else if (status === 'false') {
+      const currentlyReadBooks = books.filter(book => book.finished === false)
+      setFilteredBooks(currentlyReadBooks)
+    } else if (status === '') {
+      setFilteredBooks(books)
+    }
```

Comment:
Sometimes it is nicer to not use `else if` but only `if` statements. For better readability:
```javascript
if(status === 'finishedBooks') {
  ...
}

if(status === 'currentlyReading') {
  ...
}

if(status === '') {
  ...
}
```